### PR TITLE
Added sewtmc.is-not.cool

### PR DESCRIPTION
**Pull request by SkyExploreWasTaken**
Just to play Minecraft servers without a .ATERNOS.org domain

### CODE REVIEW

**Review by CuteDog5695 — APPROVED** [review]
Use case seems good to me, will merge

**Comment by SkyExploreWasTaken** [comment]
thx

**Comment by SkyExploreWasTaken** [comment]
huh

File: domains/sewtmc.is-not.cool.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "description": "Aternos Domain for myself",
+    "domain": "is-not.cool",
+    "subdomain": "sewtmc",
+
+    "owner": {
+        "username": "SkyExploreWasTaken",
+        "email": "contact@sewt.is-a.dev",
```
Comment (CuteDog5695) [inline domains/sewtmc.is-not.cool.json:8]:
```suggestion
        "email": "contact@sewt.is-a.dev"
```

**Review by CuteDog5695 — APPROVED** [review]

**Comment by SkyExploreWasTaken** [comment]
letsgo !!

**Pull request merged**